docs/guide_restli_server.md

```diff
@@ -1295,6 +1288,51 @@ FortunesBuilders().partialUpdate().id(1L).input(patch).build();
 `PatchGenerator.diff(original, revised)` can also be used to create a
 minimal partial update.
 
+#### Returning entity in PARTIAL_UPDATE response
+
+By default, the patched entity is not returned in the PARTIAL_UPDATE response because
+the client already has the patch data and possibly has the rest of the entity as well.
+However, there are use cases where the server will attach additional data to the new entity or the user
+simply doesn't have the whole entity. Returning the entity in the PARTIAL_UPDATE response saves  the client
+another GET request.
+
+Starting in Rest.li version 23.0.20 or 24.0.0 (?), we provide the developer the option to
```

Comment:
Can we explain how to prevent the entity to be returned if a query param is passed?